### ISSUE TITLE
aegisub subtitle format for extension not found

### ISSUE THREAD

**Issue by hypertextPreprocessor** [Aegisub/Aegisub#347]
linux (ubuntu 22.04)
I use appImage form this repos
while I export the srt format subtitle ,it shows this:
![image](https://github.com/Aegisub/Aegisub/assets/28087242/f08278a0-89f8-41cc-89c6-bb32bffc7335)

**Comment by RepeatedKibbles** [comment]
You might need to manually add .srt after name file.

**Comment by slrslr** [comment]
Yes, it worked, but when i select .srt in a save window drop-down menu, it should use the .srt extension automatically, i think that it works like that in numerous apps. And the error message is not clear enough.
